sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -402,8 +404,11 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
           std::string const& componentName,
           std::string const& componentVersion,
           TelemetryOptions options = TelemetryOptions())
-          : m_telemetryId(BuildTelemetryId(componentName, componentVersion, options.ApplicationId))
-      {
+          : m_telemetryId(BuildTelemetryId(
+              componentName,
+              componentVersion,
+              options.ApplicationId)) // LCOV_EXCL_START
+      { // LCOV_EXCL_STOP
```

Comment:
It's hard to see when we'd use such a pattern. Starting/stopping with just one line, which happens to be a brace, is weird :(

Reply:
Gcovr false positives, or the compiler-generated code that we don't see/have no ability to change :(